docs/book/src/user/quick-start.md

```diff
@@ -733,8 +733,8 @@ spec:
 
 After the controlplane is up and running, let's retrieve the [target cluster] Kubeconfig:
 
-{{#tabs name:"tab-getting-kubeconfig" tabs:"AWS|Azure|GCP|vSphere|OpenStack, Docker"}}
-{{#tab AWS|Azure|GCP|vSphere|OpenStack}}
+{{#tabs name:"tab-getting-kubeconfig" tabs:"AWS|Azure|GCP|vSphere|OpenStack|Docker"}}
+{{#tab AWS|Azure|GCP|vSphere|OpenStack|Docker}}
```

Comment:
You mentioned in the description:
>  get kubeconfig section doesn't have openstack but actually openstack has same requirements to other platform

It seems like you are creating a single tab for all platforms. That is, you are merging the `Docker` instructions into the tab for ` AWS|Azure|GCP|vSphere|OpenStack`. 

I'm not sure how this fulfills the intent of this PR which you describe as updating OpenStack documentation.

Reply:
you are right, I believe I used wrong version I used...